### PR TITLE
Part 2 of Message Boards

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/configuring-a-message-boards-category-mailing-list.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Configuring a Message Boards Category Mailing List
+
+This article documents how to configure a mailing list for a message board category. Liferay DXP supports both POP and IMAP protocols. (The instructions below use POP. For more information, see [Read Gmail messages on other email clients using POP](https://support.google.com/mail/answer/7104828?hl=en))
+
+To add a mailing list to a Message Boards Category:
+
+ > Note that the steps below apply to updating an existing _Category_ which are virtually the same when they first create the _Category_.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/configuring-a-message-boards-category-mailing-list.md]: Please replace w/ new admonition note syntax.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/README.rst [hunk 2]
```diff
@@ -1,9 +1,9 @@
-Message Boards
-==============
+User Guide
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/README.rst]: Add the rest of the articles here following the pattern laid out.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/message-boards-mailing-lists-overview.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Message Boards Mailing Lists Overview
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/message-boards-mailing-lists-overview.md]: This article should be condensed into `Configuring a Message Boards Category Mailing List` and removed.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Moderating Message Boards
+
+This article documents the features available for site administrators or moderator roles to facilitate day-to-day thread administration. See the [Message Boards Permissions Reference](./08-message-boards-permissions-reference.md) for more information on assigning _Message Boards_ permissions to a role. To learn more about the generic Roles and Permissions, see [Roles and Permissions](https://help.liferay.com/hc/articles/360017895212-Roles-and-Permissions).
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: I would reword this introduction for several reasons:

`This article documents...` - is redundant to the title of the article. - instead of that, I would just write something to the effect if:

DXP's Message Boards include features that enable effective moderation. This includes thread locking, thread moving and merging, user bans, and thread flagging.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Here are some features:
+
+* [Locking Threads](#locking-threads)
+* [Moving Threads](#moving-threads)
+* [Banning Users](#banning-users)
+* [Reviewing Flagged Threads](#reporting-flagged-threads)
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: Remove this entirely.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## Locking Threads
+
+Site administrators can halt activity on a thread by selecting _Lock_ from the thread's Actions menu. Once a thread is locked, no one can reply to the thread.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: `no one can reply to the thread until it is unlocked by a moderator.`

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+1. Click _Lock_.
+
+    ![Locking a Thread](./moderating-message-boards/images/02.png)
+
+To unlock a thread:
+
+1. Click (![Actions](./moderating-message-boards/images/07.png)) next to the thread to be unlocked.
+1. Click _Unlock_.
+
+When a thread is unlocked, forum members can resume all thread activities.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: I think this is unnecessary to say.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+    ![Moving a Thread](./moderating-message-boards/images/03.png)
+
+1. Click _Move_ to move the thread.
+
+The thread is now in the new category.
+
+## Banning Users
+
+Unfortunately, message board users can be abusive. In this case, moderators can ban the user from the message boards. To ban a user:
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: `Unfortunately, message board users can be abusive.` Unnecessary to say.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
+# Scoping Your Message Boards
+
+Forum administrators can create and manage message boards in the Global, Site, and Page [scopes](https://help.liferay.com/hc/articles/360028819992-Widget-Scope). The following sections show administrators how to use this menu to manage message boards in each of these scopes.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md]: `The following sections show administrators how to use this menu to manage message boards in each of these scopes.`

What menu are you referring to?

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+    ![Figure 1: Select the Global site from the site selector.](./scoping-your-message-boards/images/03.png)
+
+1. Select _Message Boards_ from the _Content & Data_ menu. Any changes made here apply to the global scope.
+
+    ![Figure 2: After changing to the global scope, select Message Boards from the Content & Data menu in Site Administration.](./scoping-your-message-boards/images/04.png)
+
+## Site-scoped Message Boards
+
+By default, the _Message Boards_ app in _Site Administration_ is scoped to the current Site. This means message boards from a site about the _Lunar Resort_ does not display on a site about raising tropical fish.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md]: Unnecessary to use this analogy when we can simply say: Message Boards scoped to a particular site are not visible when viewing Messages Boards that may be deployed on another site in the same instance.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+1. Click the _Scope_ tab.
+1. Select the page's name or _PageName (Create New)_ if the page scope doesn't exist yet.
+
+   ![Figure 4. Page-scoped content](./scoping-your-message-boards/images/07.png)
+
+1. Click _Save_ then close the dialog window.
+
+![Figure 5: Page-scoped content](./scoping-your-message-boards/images/08.png)
+
+A new page scope has been created. Because this is a new page scope, there is no content just yet. Any future message boards content that is created on this page will be displayed only here.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md]: If someone walked through these steps...did they create a new page scope? I don't think so. They configured their message boards widget to be scoped to the page.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
+# Subscribing to a Message Board
+
+Subscribing to a message board is a great way to receive notifications about the latest updates. In Liferay DXP, forum members are automatically subscribed on all the threads they created.
+
+This article documents how to subscribe to a message board.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: Redundant.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
+# Subscribing to a Message Board
+
+Subscribing to a message board is a great way to receive notifications about the latest updates. In Liferay DXP, forum members are automatically subscribed on all the threads they created.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: "Users can receive notifications (email? in product? be specific) by subscribing to various aspects of a message board. By default, users are automatically subscribed to any threads they create and can opt in to subscribe to categories, specific threads via email or RSS feed."

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+To subscribe to a thread or category:
+
+1. On the _Message Boards_ widget (on a site page), click the _Actions_ icon next to a Category or Thread (in this example, _Category 1_)
+1. Click _Subscribe_.
+
+![Subscribing to a Category](./subscribing-to-a-message-board/images/01.png)
+
+The user has now subscribed to _Category 1_. The user will be notified of any changes, such as new threads and subcategories being created under _Category 1_.
+
+To see which categories and threads you have subscribed to:
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: This is a different action from subscribing to a thread - consider making it an H3 or an admonition sidebar?

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+
+1. On the _Message Boards_ widget (on a site page), click _My Subscriptions_.
+
+## Subscribing using RSS Feeds
+
+Message Boards can be published as [RSS](https://en.wikipedia.org/wiki/RSS) feeds. RSS for _Message Boards_ are enabled by default and Liferay DXP supports the following formats: RSS 1.0, RSS 2.0, or Atom 1.0.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: "Liferay DXP supports the following formats: RSS 1.0, RSS 2.0, or Atom 1.0."

Simplify wording:

"Liferay DXP supports RSS 1.0, RSS 2.0, and Atom 1.0."

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+
+![Subscribing to a Category](./subscribing-to-a-message-board/images/01.png)
+
+The user has now subscribed to _Category 1_. The user will be notified of any changes, such as new threads and subcategories being created under _Category 1_.
+
+To see which categories and threads you have subscribed to:
+
+1. On the _Message Boards_ widget (on a site page), click _My Subscriptions_.
+
+## Subscribing using RSS Feeds
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: Should this header read "Configuring an RSS Feed" - since the actual actions listed out are about configuring, not actually subscribing?

**Comment by jrhoun** [comment]
Please review my feedback and re-base/re-send. Thank you.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/configuring-a-message-boards-category-mailing-list.md (hunk 1 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/configuring-a-message-boards-category-mailing-list.md]: Rev9sed

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md (hunk 4 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: Revised

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md (hunk 5 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: Removed

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md (hunk 8 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: Removed

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md (hunk 7 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/moderating-message-boards.md]: Shortened the sentence.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md (hunk 15 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: Revised

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md (hunk 14 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: Converted to H3

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md (hunk 9 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md]: Revised.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md (hunk 10 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md]: Revised.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md (hunk 12 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: Revised

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md (hunk 11 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/scoping-your-message-boards.md]: Revised

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/message-boards-mailing-lists-overview.md (hunk 3 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/message-boards-mailing-lists-overview.md]: Consolidated

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/README.rst (hunk 2 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/README.rst]: Added the new articles

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md (hunk 16 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: I removed this because people will be RSS feeds from other assets in LR. Because RSS is declining, it was partially deprecated in WEM but is still around in Message Boards.

File: docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md (hunk 13 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/message-boards/user-guide/subscribing-to-a-message-board.md]: Revised
